browser/src/Services/Configuration/DefaultConfiguration.ts

```diff
@@ -180,6 +180,13 @@ const BaseConfiguration: IConfigurationValues = {
 
     "statusbar.enabled": true,
     "statusbar.fontSize": "0.9em",
+    "statusbar.priority": {
```

Comment:
I like you specified this in the configuration. Looks good! I left a comment below that perhaps with this, we don't need to specify the priority anymore with `createItem`? It seems like right now, there need to be two ids associated with a statusbar item - one in the configuration, and a separate one we pass to `Oni.statusbar.createItem` - I wonder if we could consolidate those?